.gitignore

```diff
@@ -0,0 +1,8 @@
+.DS_Store
+/.build
+/Packages
+xcuserdata/
+DerivedData/
+.swiftpm/configuration/registries.json
```

Comment:
maybe we want to ignore all .switfpm contents instead?
i.e. if they are user preferences we should

Reply:
I'm not actually sure when or why I added .swiftpm to the .gitignore, but in general it seems like folks _do_ commit it, as it contains things like an `xcworkspace` and references to specific versions of tools, so I'll be removing this from the .gitignore.